src/main/java/com/uid2/admin/vertx/service/ClientKeyService.java

```diff
@@ -72,8 +72,13 @@ public void setupRoutes(Router router) {
                 this.handleRewriteMetadata(ctx);
             }
         }, Role.CLIENTKEY_ISSUER));
+
         router.get("/api/client/list").handler(
                 auth.handle(this::handleClientList, Role.CLIENTKEY_ISSUER));
+
+        router.get("/api/client/listBySite/:siteId").handler(
+                auth.handle(this::handleClientListBySite, Role.CLIENTKEY_ISSUER));
```

Comment:
I think we want `Role.SHARING_PORTAL` as well?

Reply:
Ah yeah we would sorry.